### PR TITLE
feat(scripts): update deploys scripts 

### PR DESCRIPTION
**Pull request by P-Yevhenii**

### CODE REVIEW

File: scripts/tests/test_router.ts [hunk 1]
```diff
@@ -21,20 +21,20 @@ async function main() {
...
 
-    // 2. Provide LUNA-UST liquidity
+    // 2. Provide LunaUst liquidity
     await provideLiquidity(network, astroport, cl.wallet.key.accAddress, network.poolLunaUst, [
         new NativeAsset('uluna', liquidity_amount.toString()),
         new NativeAsset('uusd', liquidity_amount.toString())
     ])
 
-    // 3. Fetch the pool balances
+    // 3. Pools balance
```
Comment (ShadoySV) [inline scripts/tests/test_router.ts:37]: Keep these above as they were

File: scripts/tests/tests.ts [hunk 2]
```diff
@@ -18,13 +18,13 @@ async function main() {
...
 
-    // 2. Stake ASTRO
+    // 2. Deposit ASTRO to staking
     await stake(network, astroport, wallet.key.accAddress)
 
     // 3. Swap tokens in pool
     await swap(network, astroport, wallet.key.accAddress)
 
-    // 4. Collect Maker fees
+    // 4. Maker collect fees
```
Comment (ShadoySV) [inline scripts/tests/tests.ts:27]: And these

File: scripts/tests/tests.ts [hunk 3]
```diff
@@ -38,7 +38,7 @@ async function provideLiquidity(network: any, astroport: Astroport, accAddress:
     const liquidity_amount = 100000000;
     const pool_uust_astro = astroport.pair(network.poolAstroUst);
 
-    // Provide liquidity in order to swap
+    // Provide liquidity to swap
```
Comment (ShadoySV) [inline scripts/tests/tests.ts:41]: And this

File: scripts/build_artifacts.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```
Comment (ShadoySV) [inline scripts/build_artifacts.sh:1]: This file does the same as ```build_release.sh```

**Review by ShadoySV — CHANGES_REQUESTED** [review]

**Comment by epanchee** [comment]
Make build_{app,env,schema}.sh scripts executable (`chmod +x`)

**Pull request closed without merging**